runtime/src/in_mem_accounts_index.rs

```diff
@@ -264,6 +271,42 @@ impl<T: IndexValue> InMemAccountsIndex<T> {
         result
     }
 
+    pub(crate) fn flush(&self) {
+        let flushing = self.flushing_active.swap(true, Ordering::Relaxed);
+        if flushing {
+            // already flushing in another thread
+            return;
+        }
+
+        self.flush_internal();
+
+        self.flushing_active.store(false, Ordering::Relaxed);
+    }
+
+    fn flush_internal(&self) {
+        let was_dirty = self.bin_dirty.swap(false, Ordering::Relaxed);
```

Comment:
Same comment here as with `flushing_active`. I think this should be `Ordering::Acquire`.

...But where does `bin_dirty` get set to true? That one should be `Ordering::Release`.

Reply:
done, and added set_bin_dirty, with Release.